sdk-extension/opentelemetry-sdk-extension-aws/tests/trace/test_aws_xray_ids_generator.py

```diff
@@ -15,28 +15,46 @@
 import datetime
 import time
 import unittest
+import sys
+import os
 
-from opentelemetry.sdk.extension.aws.trace import AwsXRayIdGenerator
 from opentelemetry.trace.span import INVALID_TRACE_ID
 
+# Make sure we import local file instead of /site-packages
+_src_folder_path = os.path.dirname(__file__).split('/')[0:-2]
+_aws_xray_file = os.path.join('/'.join(_src_folder_path), 'src')
+sys.path.append(_aws_xray_file)
+
+from opentelemetry.sdk.extension.aws.trace import AwsXRayIdGenerator
+
 
 class AwsXRayIdGeneratorTest(unittest.TestCase):
-    def test_ids_are_valid(self):
+    def test_trace_ids_are_valid(self):
         id_generator = AwsXRayIdGenerator()
         for _ in range(1000):
             trace_id = id_generator.generate_trace_id()
             self.assertTrue(trace_id != INVALID_TRACE_ID)
             span_id = id_generator.generate_span_id()
             self.assertTrue(span_id != INVALID_TRACE_ID)
 
-    def test_id_timestamps_are_acceptable_for_xray(self):
+    def test_trace_id_timestamps_are_acceptable_for_xray(self):
         id_generator = AwsXRayIdGenerator()
         for _ in range(1000):
             trace_id = id_generator.generate_trace_id()
```

Comment:
I think this test could be improved to something like

```python
before_time = int(time.time())
trace_id = id_generate_trace_id()
trace_id_time = trace_id >> 96
after_time = int(time.time())
self.assertGreater(trace_id_time, before_time)
self.assertLessEqual(trace_id_time, after_time)
```

but given this runs 1000 times we'd expect the test to already be checking the timestamp reasonably well.